lib/logger/index.spec.ts

```diff
@@ -160,13 +160,14 @@ describe('logger/index', () => {
       constructor(public field: string) {}
     }
 
+    const prBody = 'test';
```

Comment:
> * logger related, in `logger/utils` `sanitizeValue` - [prBody is no longer sanitized](https://github.com/renovatebot/renovate/issues/16216#issuecomment-1165223522).

This sounds important. Is it working with this PR again? Or do we need to do more to make sure things are properly sanitized?

Reply:
Yes, what does this mean?

Reply:
we sanitize all string values, see line 105